var/spack/repos/builtin/packages/gxsview/package.py

```diff
@@ -26,6 +26,7 @@ class Gxsview(QMakePackage):
     depends_on("qt@5.14.0:+opengl+gui")
     depends_on("vtk@8.0:+qt+opengl2")  # +mpi+python are optional
     conflicts("%gcc@:7.2.0", msg="Requires C++17 compiler support")  # need C++17 standard
+    patch("gcc11.patch", when="@2021.07.01 %gcc@11:")
```

Comment:
Can you add a brief comment describing the patch above the directive?

Reply:
Yes, done.